### ISSUE TITLE
Is there a method for bottomtabwillselectedlistener？

### ISSUE THREAD

**Issue by jeffy7** [wix/react-native-navigation#5599]
My question isn't Issue.

My requirement is that if I click a tab without logging in, I will push it directly, or present a page, and I can't see the tab's page. Now, if I turn off the animation of push or present, there will still be a flash. I use the method of registerbottom tabselectedlistener, but only after the tab is selected. Monitor. Is there any monitor before being selected?

Or there are other ways.

**Comment by pontusab** [comment]
Related https://github.com/wix/react-native-navigation/issues/5545

**Comment by ItsNoHax** [comment]
Closing this in favour of #5545
